api/groups/addressGroup.go

```diff
@@ -28,7 +28,8 @@ const (
 	getESDTNFTDataPath        = "/:address/nft/:tokenIdentifier/nonce/:nonce"
 )
 
-type addressFacadeHandler interface {
+// AddressFacadeHandler defines the methods to be implemented by a facade for handling address requests
+type AddressFacadeHandler interface {
```

Comment:
why do we need all these interfaces to be exported?

Reply:
done